flytekit/image_spec/image_spec.py

```diff
@@ -26,38 +26,40 @@ class ImageSpec:
         builder: Type of plugin to build the image. Use envd by default.
         source_root: source root of the image.
         env: environment variables of the image.
-        registry: registry of the image.
+        output_registry: registry to push the final image.
```

Comment:
Can we not rename it? It will break existing usres

Reply:
Sure I'll change it. I renamed it because I thought having a "registry" and "private_registries" parameter may cause confusion.